hw/top_earlgrey/dv/chip_mask_rom_tests.hjson

```diff
@@ -7,29 +7,28 @@
 
   # Note: Please maintain alphabetical order.
   tests: [
-    // Mask ROM func tests to be run with test ROM.
+    // Signed chip-level tests to be run with mask ROM, instead of test ROM.
     {
-      name: chip_sw_mask_rom_keymgr_functest
+      name: chip_sw_uart_smoketest_signed
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["sw/device/silicon_creator/testing/sw_silicon_creator_lib_driver_keymgr_functest:1"]
-      en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+sw_test_timeout_ns=5000000"]
+      sw_images: ["sw/device/tests/uart_smoketest:1:signed"]
+      en_run_modes: ["sw_test_mode_mask_rom"]
     }
 
-    // Chip-level tests to be run with mask ROM, i.e., ROM_EXT tests.
-    // Note, these tests will be signed with test keys, prior to loading.
+    // Mask ROM func tests to be run with test ROM.
     {
-      name: chip_sw_mask_rom_uart_smoketest_signed
+      name: mask_rom_keymgr_functest
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["sw/device/tests/rom_ext_uart_smoketest:1:signed"]
-      en_run_modes: ["sw_test_mode_mask_rom"]
+      sw_images: ["sw/device/silicon_creator/testing/sw_silicon_creator_lib_driver_keymgr_functest:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=5000000"]
     }
   ]
 
   regressions: [
     {
-      name: mask_rom_functest
-      tests: ["chip_sw_mask_rom_keymgr_functest"]
+      name: mask_rom_functests
+      tests: ["mask_rom_keymgr_functest"]
```

Comment:
Add `chip_sw_uart_smoketest_signed` to this list too?

Reply:
I think it makes sense to create a separate regression for the signed chip-level tests run with mask ROM, as mask ROM functests are completely different (they run with the test rom). Also note that this test won't pass until #10863 is resolved.

Fixed.

Reply:
DVSim adds these default regression targets `all`, `nightly`, `smoke`, `v1|2|2s|3` and `all_once` for all designs automatically. Users can add custom regression targets for convenience, which allows them to group a set of tests and run them in one shot. This is fine by me, assuming that it is useful to you.